gbpservice/nfp/core/controller.py

```diff
@@ -522,7 +522,12 @@ def load_nfp_modules(conf, controller):
         modules_dir = base_module.__path__[0]
         try:
             files = os.listdir(modules_dir)
-            for pyfile in set([f for f in files if f.endswith(".py")]):
+            pyfiles = set([f for f in files if f.endswith(".py")])
+            for pyfile in pyfiles:
+                module_name = pyfile.strip('.py')
+                nsd_module_name = module_name + '_NSD.py'
```

Comment:
we want to skip _NSD.py here ?
can this file be put outside of 'modules/' folder ?